config/ftbquests/quests/chapters/tools.snbt

```diff
@@ -3245,58 +3356,72 @@
 			title: "Powered by Mysterious Energy called..."
 			x: -6.5d
 			y: -17.5d
-			subtitle: "RF? This is mysterious? Platinum is mysterious pff"
-			description: [
-				"Fourth tier is made of Diamonds and Platinum. Once again, Tier below is needed to get Tier 4."
-				""
-				"Even tho Platinium is much more expensive than Diamond, it is only a tiny bit better in terms of speeds than it."
-			]
+			subtitle: "RF? This is mysterious? Wasn't RF just FE..."
+			description: ["Fourth tier is the high-end of your upgrade path, giving you a choice of 2 jetpacks from premium materials. Which one you should choose?"]

```

Comment:
`Which one you should choose?` -> `Which one should you choose?`

`Which one will you choose?` makes a bit more sense though, imo